config/processors/syslog_log_security_layer7.securespan.soa.gw.conf

```diff
@@ -5,7 +5,16 @@ input {
     address => VAR_PIPELINE_NAME
   }
 }
+
 filter {
+  # Parsing the syslog pri header if present
+  grok {
+    match => { "message" => "^.*?<(?<pri>\d+)>(\s)?(?<actual_msg>.*)" }
+  }
+  syslog_pri {
+  syslog_pri_field_name => "pri" 
+  remove_field => [ "pri" ]
```

Comment:
we are trying to compare pri in the later code, so dont remove it here, pri can be removed at end